docs/source/posts/2023/2023-06-12-week-2-joaodellagli.rst

```diff
@@ -0,0 +1,86 @@
+Week 2: The Importance of (good) Documentation
+==============================================
+
+.. post:: June 12, 2023
+   :author: João Victor Dell Agli Floriano
+   :tags: google
+   :category: gsoc
+
+
+Hello everybody, welcome to the week 2 of this project! I must admit I thought this would be simpler than it is currently being, but I forgot that when it comes to dealing with computer graphics' applications, things never are. Below, some updates on what I have been up to for this past week. 
+
+This Last Week's Effort
+-----------------------
+
+Last week, I was facing some issues with a VTK feature essential so I could move forward with my project: Framebuffer Objects. 
+As described in my `last blogpost <https://blogs.python-gsoc.org/en/joaodellaglis-blog/the-fbo-saga-week-1/>`_, for some reason the 2D allocation methods for it weren't working.
```

Comment:
it would be better to do a local reference. In rst, you can do a reference to another file and link will appears during the build. 

Internal link have a higher priority than external link.